src/lib/code-sample-editor.ts

```diff
@@ -8,20 +8,23 @@ import {
   query,
   queryAll,
 } from 'lit-element';
-import {until} from 'lit-html/directives/until';
-import {FileRecord, CodeSampleEditorTextarea, RemoteSw} from './types';
-import {EMPTY_INDEX} from './constants';
+import {until} from 'lit-html/directives/until.js';
```

Comment:
Google style guide says leave this off. Do we care? will this ever be used internally? (IK it'd be third_party, but being able to work on it and work towards TS readability is appreciated)

Reply:
Google style guide is workable internally where the files are never directly loaded into browsers, but the only way to get these to work natively in browsers right from the tsc output is to include the file extensions. You may see things working because es-dev-server will do the node resolution even for relative paths, but that's relying on a specific and non-standard tool. If you happen to use extension-less imports in MWC, I'd change those for maximum compatibility externally.